src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -618,6 +615,7 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
               case Some(ts: Instant) => Timestamp.from(ts)
               case Some(x)           => x
```

Comment:
I wonder if we have a case where a column is an `Option[Map[String,Any]]`, won't we need to deserialize this as Json as well?

Reply:
More generally, and this isn't directed at this PR per se, but I think this `match` block needs a little work to make it more robust. For instance, we only handle `Option[Instant]`, but not required `Instant`. Seems like we need a recursive match here to handle this case, as well as the one I raised in the previous comment, which is directed at this PR.